<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="rb_ac">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_dc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_ohms">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OHMS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_get">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_result">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_testm">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>280</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>DMM Test</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_timer">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>104</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (sec)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
